content/types/_message.textile

```diff
@@ -17,6 +17,15 @@ h4.
 - <div lang="objc">data</div> := The message payload, if provided<br>__Type: @NSString *@, @NSData *@, @NSDictionary *@, @NSArray *@__
 - <div lang="swift">data</div> := The message payload, if provided<br>__Type: @String@, @NSData@, @Dictionary@, @Array@__
 
+- <div lang="jsall">extras</div> := Metadata and/or ancillary payloads, if provided<br>__Type: @JSON Object@__
+- <div lang="java">extras</div> := Metadata and/or ancillary payloads, if provided<br>__Type: @JSONObject@, @JSONArray@__
+- <div lang="csharp">Extras</div> := Metadata and/or ancillary payloads, if provided<br>__Type: plain C# object that can be converted to Json__
+- <div lang="ruby">extras</div> := Metadata and/or ancillary payloads, if provided<br>__Type: @Hash@, @Array@__
+- <div lang="python">extras</div> := Metadata and/or ancillary payloads, if provided<br>__Type: @Dict@, @List@__
+- <div lang="swift">extras</div> := Metadata and/or ancillary payloads, if provided<br>__Type: @Dictionary@, @Array@__
+- <div lang="objc">extras</div> := Metadata and/or ancillary payloads, if provided<br>__Type: @NSDictionary *@, @NSArray *@__
+- <div lang="php">extras</div> := Metadata and/or ancillary payloads, if provided<br>__Type: @Associative Array@, @Array@__
```

Comment:
Same description copied 8 times? This can be all one line, something like
`<span lang="csharp">Extras</span><span lang="default">extras</span := Metadata and/or ancillary payloads, if provided<br>__Type: <span lang="jsall">@Object@</span><span lang="java">... etc.`